src/main/java/ingest/utility/IngestUtilities.java

```diff
@@ -438,4 +446,28 @@ public void deleteDatabaseTable(String tableName) throws IOException {
 			postGisStore.dispose();
 		}
 	}
+	
+	/**
+	 * Private method to post requests to elastic search for deleting the service metadata.
+	 * 
+	 * @param DataResource
+	 *            Data object
+	 * @param url
+	 *            Elasticsearch endpoint for deletion
+	 * @return PiazzaResponse response 
+	 */
+	public PiazzaResponse deleteElasticsearchByDataResource(DataResource dataResource, String url) {
+		try {
+			HttpHeaders headers = new HttpHeaders();
+			headers.setContentType(MediaType.APPLICATION_JSON);
+			HttpEntity<DataResource> entity = new HttpEntity<DataResource>(dataResource, headers);
+			return restTemplate.postForObject(url, entity, PiazzaResponse.class);
+		} catch (Exception exception) {
```

Comment:
Before you catch a generic Exception, do a "catch (HttpClientErrorException | HttpServerErrorException httpException)" so that you can log httpException.getResponseBodyAsString() and httpException.getStatusCode() for more explicit logs.